Public/Src/Deployment/buildXL.dsc

```diff
@@ -22,6 +22,7 @@ namespace BuildXL {
             importFrom("BuildXL.Tools").Execution.Analyzer.exe,
             importFrom("BuildXL.Tools").BxlScriptAnalyzer.exe,
             importFrom("BuildXL.Cache.VerticalStore").Analyzer.exe,
+            importFrom("BuildXL.Tools").Xldb.Analyzer.exe,

```

Comment:
>importFrom("BuildXL.Tools").Xldb.Analyzer.exe, [](start = 12, length = 46)

This is more like a demo than a tool that we want to distribute. Let's put it in out\bin\{release/debug}\tools instead of main deployment

Reply:
Could you clarify what you mean by this? The exe is generated in out/bin/debug/tools 

---
In reply to: [316292295](https://github.com/microsoft/BuildXL/pull/775#discussion_r316292295) [](ancestors = 316292295)